### PR TITLE
Revert "CDK: erigon docs"

### PR DESCRIPTION
**Pull request by kmurphypolygon**
Reverts 0xPolygon/polygon-docs#440

### CODE REVIEW

File: docs/cdk/concepts/rollup-vs-validium.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ Rollups use Ethereum as a [data availability](https://docs.polygon.technology/cd
 
 Using Ethereum to store transaction data is generally considered the most secure option for DA as it leverages Ethereum's security and decentralization. However, this approach is costly, as the L2 must pay Ethereum&rsquo;s high gas fees for storing data on the L1, which typically results in higher gas fees for users.
 
-Within the rollup category, there are further nuances to storing transaction data on Ethereum. Some rollups post serialized transaction data directly, whereas others post state differences instead. Some rollups use [calldata](https://docs.soliditylang.org/en/v0.8.26/types.html#data-location) to store transaction data, while others use more recent Ethereum features such as blobs, introduced in [EIP-4844](https://www.eip4844.com/).
+Within the rollup category, there are further nuances to storing transaction data on Ethereum. Some rollups post serialized transaction data directly, whereas others post state differences instead. Some rollups use [calldata](https://docs.soliditylang.org/en/v0.8.26/types.html#data-location) to store transaction data, while others use more recent Ethereum features such as Blobs, introduced in [EIP-4844](https://www.eip4844.com/).
```
Comment (EmpieichO) [inline docs/cdk/concepts/rollup-vs-validium.md:11]: It's better left as "blobs" ... and not capitalised, in the same way that we do with "blocks" and "batches".

File: docs/cdk/concepts/rollup-vs-validium.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ Rollups use Ethereum as a [data availability](https://docs.polygon.technology/cd
...
 
 The CDK provides full flexibility to developers to choose what to do with transaction data, including the ability to build rollups that store data on Ethereum as a rollup like the Polygon zkEVM. 
 
 !!! note
-    Currently, the rollup mode of Polygon CDK does not support blobs (EIP-4844), but this functionality is coming soon.
+    Currently, the rollup mode of Polygon CDK does not support BLOB mode (EIP4844), but this functionality is coming soon.
```
Comment (EmpieichO) [inline docs/cdk/concepts/rollup-vs-validium.md:16]: It's again better to leave it as "blobs (EIP-4844)" ... than to change to "BLOB mode (EIP4844)". 
Talking about "BLOB mode" may be confusing especially after mentioning "the rollup mode", ... "blob" refers to the formatting of data, more like 'blocks' and 'batches'.
About EIPs, the standard is "EIP-_ _ _ _ "

**Review by EmpieichO — APPROVED** [review]

File: docs/cdk/concepts/rollup-vs-validium.md (hunk 2 above)
Comment (kmurphypolygon) [reply docs/cdk/concepts/rollup-vs-validium.md:16]: it's strange, these were removed weeks ago ... i'm checking to see what happened

**Comment by kmurphypolygon** [comment]
Closing as too many unwanted changes coming in.

**Pull request closed without merging**